src/Data/MinLen.hs

```diff
@@ -480,3 +481,13 @@ minimumBy :: MonoFoldable mono
           -> Element mono
 minimumBy cmp = minimumByEx cmp . unMinLen
 {-# INLINE minimumBy #-}
+
+-- MonoComonad instance for IsSequence mono => NonNull mono
+instance IsSequence mono => MonoComonad (MinLen (Succ Zero) mono) where
```

Comment:
Instead of `Succ Zero`, `Succ a` would allow this to work for arbitrarily sized `MinLen`s.